### PR TITLE
Release1 functions

### PR DESCRIPTION
**Pull request by ritika2**
implemented below functions for release 1
is_lift_empty function 
moving_lift_up function
max_waiting_time function
average_waiting_time function

### CODE REVIEW

**Review by divyalakshmi-padmanaban — APPROVED** [review]
looks fine to me

File: include/average_waiting_time.h [hunk 1]
```diff
@@ -0,0 +1,15 @@
+/*@file average_waiting_time.h
```
Comment (MaazJamal) [inline include/average_waiting_time.h]: doxygen comment start with /**

File: include/average_waiting_time.h [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+ *@brief Has the declaration for the average_waiting_time function
+ *@detail The function has one input parameter: time_difference which is the difference between
+ * the time passenger arrived and the time he got into the elevator
+ *@return float returns the average time taken by the passengers waiting to get into the elevator
+ */
+
+#ifndef AVERAGE_WAITING_TIME_H_INCLUDED
```
Comment (MaazJamal) [inline include/average_waiting_time.h:8]: usually the coding convention is to use __AVERAGE_WAITING_TIME_H__ as header gurad name. This is a small issue.

**Review by MaazJamal — COMMENTED** [review]
The code is correct. However the doxygen comments need an extra * at start like this /**
The header guards have been named in an unconventional manner.
You have also deleted src/count_passenger.c and src/input_floors.c in your local repository. You need to add them back to the local repository before pushing the changes again. Also there are a few conflicts. that need to be resolved. Backup your code before trying out the suggested fix. As you may lose your work.

**Review by MaazJamal — APPROVED** [review]
Okay the conflicts have been resolved i think we can merge safely

**Review by Hariprasadm1408 — APPROVED** [review]
Looks good

**Comment by ritika2** [comment]
Merging the pull request after approvals of the reviewers.

**Pull request merged**